docs/baseband/mark4/index.rst

```diff
@@ -39,7 +39,8 @@ can be calculated (for longer files, this can be calculated from the file)::
     ...                 frames_per_second=400)
     >>> fh
     <Mark4StreamReader name=... offset=0
-        samples_per_frame=80000, nchan=8, frames_per_second=400, bps=2,
+        frames_per_second=400, samples_per_frame=80000,
+        sample_shape=SampleShape(nchan=8), bps=2,
```

Comment:
The example makes me wonder whether we should call these `Mark4Shape`, etc... (as users cannot initialize these themselves...) But maybe leave that for another PR.

Reply:
I played with the idea, but didn't like it because there's nothing about the sample shape that makes it unique to Mark 4.  Even for a single file, the shape could be different between squeezed and non-squeezed.